src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheck.java

```diff
@@ -284,7 +284,9 @@ public void visitToken(DetailAST ast) {
         final TextBlock cmt =
             contents.getJavadocBefore(lineNo);
         if (cmt == null) {
-            log(lineNo, MSG_MISSING_TAG, tag);
+            if (tag != null) {
```

Comment:
> Also note confusing violation message ("null" tag).

@strkkk @romani So are we not resolving this in this issue or is this already another issue?

Reply:
We can do in this PR or separate , up to PR author.

Fyi, we are very close to merge at https://github.com/checkstyle/checkstyle/pull/12945

Reply:
We can do in this PR or separate , up to PR author.

Fyi, we are very close to merge at https://github.com/checkstyle/checkstyle/pull/12945

Reply:
Sorry, my bad. Actually I fixed the message stuff but forgot about skipping javadoc. 

Now it is fixed properly. Weird message with null wont be reported as well.